include/MCU_rev15_defs.h

```diff
@@ -0,0 +1,18 @@
+#ifndef __MCU15_H__
+#define __MCU15_H__
+
+// pindefs
+const int ADC1_CS = 34;
+const HardwareSerial* STEERING_SERIAL = &Serial5;
```

Comment:
Wouldn't it be easier to just modify one file instead of a ton in various libraries? These pins are pretty arbitrary and probably different from board to board. It does not seem to make libraries any more portable if it's shoved into individual interaces, especially if you want a separate repo for the libraries. It will only make sense if we want a default pin assignment to save the user some trouble.

Reply:
I think the current implementation is the best option. If we were to change pin definitions in rev16+, it's preferable to change these things in one file + the one place it's included vs. going into a bunch of interfaces to change pin definitions. 

This file should be included in the main file and have its values passed as instantiation parameters to interfaces there.